package.json

```diff
@@ -91,7 +91,7 @@
     "commander": "^2.17.0",
     "concurrently": "^4.0.1",
     "cross-env": "^5.2.0",
-    "danger": "^3.8.4",
+    "danger": "4.0.1",
```

Comment:
@davegaeddert the caret is lost here

Reply:
True. Right now you can use:
```yml
settings:
  constraint_prefix: '^'
```

But that will apply to all dependencies in package.json. In your case it looks like they almost all use "^", but not completely... Anyway, a smarter/automatic version of this could be added here (PRs welcome): https://github.com/dependencies-io/js/blob/master/src/manifest.js#L108